roomranger/frontend/src/App.js

```diff
@@ -1,81 +1,95 @@
-import './App.css';
-import "../node_modules/bootstrap/dist/css/bootstrap.min.css"
+import "./App.css";
```

Comment:
To maintain consistency, let's all use single quotation marks.

Reply:
Done